packages/react/examples/shadow-dom/package-lock.json

```diff
@@ -9,7 +9,6 @@
       "version": "4.5.1",
       "license": "MIT",
       "dependencies": {
-        "@axe-core/react": "^4.5.1",
```

Comment:
axe-core appears to be removed rather than updated to 4.5.2

Reply:
axe-core isn't a dependency of this package

Reply:
Oh my mistake